RSSReader-Maruyama/Modules/Login/LoginViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  LoginViewController.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+import AuthenticationServices
+import FBSDKCoreKit
+import FBSDKLoginKit
+import GoogleSignIn
+
+class LoginViewController: UIViewController {
+    
+    @IBOutlet weak var loginButtonStackView: UIStackView!
+    private let SELECT_FEEDS_SEGUE_ID = "selectFeeds"
+    
+    fileprivate let userDataManager = UserDataManager()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupLoginButtons()
+    }
+    
+//MARK:- Setup SignIn Button
```

Comment:
```suggestion
    // MARK: - Setup SignIn Button

```
インデントがずれています。
またMarkコメントのあとは改行を入れましょう。